### ISSUE TITLE
posix.listdir is not found

### ISSUE THREAD

**Issue by magwas** [google/pytype#371]
I believe it is a bug.
mypy output also included, as this hints to a bug in an underlying common library
```
developer@efbaed8fbb53:/build/bar$ ls
foo.py
developer@efbaed8fbb53:/build/bar$ cat foo.py 
from posix import listdir

print(listdir("."))
developer@efbaed8fbb53:/build/bar$ python3 foo.py 
['foo.py']
developer@efbaed8fbb53:/build/bar$ mypy foo.py 
foo.py:1: error: Module 'posix' has no attribute 'listdir'
developer@efbaed8fbb53:/build/bar$ pytype foo.py 
Computing dependencies
Analyzing 1 sources with 0 local dependencies
ninja: Entering directory `/build/bar/.pytype'
[1/1] check foo
FAILED: /build/bar/.pytype/pyi/foo.pyi 
pytype-single --imports_info /build/bar/.pytype/imports/foo.imports --module-name foo -V 3.6 -o /build/bar/.pytype/pyi/foo.pyi --analyze-annotated --nofail --quick /build/bar/foo.py
File "/build/bar/foo.py", line 1, in <module>: Can't find module 'posix.listdir'. [import-error]

For more details, see https://google.github.io/pytype/errors.html#import-error.
ninja: build stopped: subcommand failed.

```

**Comment by rchen152** [comment]
Thanks for the report! It looks like the problem is that `listdir` is defined in `os` (https://github.com/python/typeshed/blob/d05a9d3d833625f251cc84f88b81afade9a95325/stdlib/3/os/__init__.pyi#L437), when it should be defined in posix and imported in os.

As a workaround, `from os import listdir` should work.

**Comment by rchen152** [comment]
Opened https://github.com/python/typeshed/pull/3258.

**Comment by rchen152** [comment]
The PR has been merged; I'll close this issue once we release a version of pytype containing the fix.

**Comment by rchen152** [comment]
Sorry for the long delay. This fix has landed in pytype 2019.12.06.
